src/app/services/baw-api/analysis/analysis-job-item-results.service.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable } from "@angular/core";
+import {
+  emptyParam,
+  filterParam,
+  IdOr,
+  id,
+  IdParamOptional,
+  option,
+  ReadonlyApi,
+  param,
+} from "@baw-api/api-common";
+import { BawApiService, Filters } from "@baw-api/baw-api.service";
+import { stringTemplate } from "@helpers/stringTemplate/stringTemplate";
+import { AnalysisJobItemResults } from "@models/AnalysisJobItemResults";
+import { AnalysisJob } from "@models/AnalysisJob";
+import { Observable } from "rxjs";
+import { AudioRecording } from "@models/AudioRecording";
+import { Resolvers } from "@baw-api/resolver-common";
+
+const analysisJobId: IdParamOptional<AnalysisJob> = id;
+const audioRecordingId: IdParamOptional<AudioRecording> = id;
+const analysisJobItemResultsPath = param;
+
+// Example of results endpoint:
+// /analysis_jobs/{analysisJob}/results/{audioRecordingId}/Towsey.Acoustic/folderA/fileA.wav
```

Comment:
```suggestion
// /analysis_jobs/{analysisJob}/results/{audioRecordingId}/{path}
```

Reply:
This comment would repeat what the string template says under, nullifying any additional content this comment will provide. If the file path example is incorrect usage of a comment, would it be more correct not to include the comment and instead rely on self documenting code?
![image](https://user-images.githubusercontent.com/33742269/204954832-b3593b45-6f07-4dd1-aabc-91df5c53891e.png)